storage/innobase/handler/ha_innodb.cc

```diff
@@ -8266,9 +8266,26 @@ ha_innobase::write_row(
 					ulonglong	increment;
 					dberr_t		err;
 
+#ifdef WITH_WSREP
```

Comment:
You could use static inline function instead as this same is on two places at least.

Reply:
Unfortunately, the section taken under #ifdef fills the variables (when WSREP is not used) so it cannot be done inline without loss of performance. As for the branch with WSREP, then the filling of these variables is already rendered in the separate function, which is called from the ha_innodb.cc twice (wsrep_thd_auto_increment_variables, which is located in the wsrep_thd.cc file).